app/src/main/dist/config/crate.yml

```diff
@@ -1,15 +1,51 @@
-######################### Crate Configuration Example #########################
+######################### Crate Configuration File ############################
 
 # The default configuration offers the ability to use Crate right away.
 # The purpose of this file is to give operators an overview of the various
 # different configuration settings. To read the full documentation go to
-# <https://docs.crate.io/>
+# <https://crate.io/docs/>
 
-# As mentioned before, you can use Crate right away. For fine-tuning consider
-# to configure Crate as you need or want it to behave.
 # Use this file to fine-tune your Crate cluster. If you have any questions
-# you are very welcome to conact us on our public Community on Slack
-# (https://crate.io/docs/support/slackin/).
+# you are very welcome to contact us on our Community Channel on Slack
+# <https://crate.io/docs/support/slackin/>.
+
+
+################################ Quick Settings ###############################
+
+# Recommended memory settings:
+# - set the environment variable CRATE_HEAP_SIZE to half of your memory
+#   (e.g. 26G, but not more than 30G).
+#   Depending on the OS the best place to do so is
+#   '/etc/default/crate' or '/etc/sysconfig/crate'
+# - disable swapping
+#bootstrap.mlockall: true
```

Comment:
I guess your intention having no space between `#` and the setting name was, that the user doesn't accidentally leave a leading space. I like the idea since I have seen this problem. Maybe we should do that throughout the whole file then?